### PR TITLE
feat: Create: 2022-07-07-perchè-container-sono-strumenti-utili-sviluppo-ser…

### PR DESCRIPTION
**Pull request by Francescopog**
…vizi.pubblici.md
@bfabio 
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Create: 2022-07-07-perchè-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md
- type must be lower-case
- type must be one of [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test]
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 00cb364045dc65c13b6aa407038a6fd221bf1080
</p>

File: _posts/it/2022-07-07-perchè-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Fabio ha illustrato i vantaggi dei container per lo sviluppo nell’ambito dei servizi pubblici, che contribuiscono a ridurre i problemi dati dalle differenze di ambienti, stanno alla base dei processi di *Continuous Integration e testing* e godono di un grande ecosistema in continua evoluzione.
+
+Come da tradizione nei nostri lab, Fabio ci ha portato velocemente dalla teoria alla pratica mostrandoci una demo di un container, usando lo strumento Docker e visualizzandone le principali caratteristiche. Insieme a lui, abbiamo usato un container già pronto, ne abbiamo creato uno nuovo e abbiamo infine combinato assieme più container.
+
+L’incontro si è chiuso con le domande dei partecipanti e con una call to action per tutti: Installa Docker o Podman e… sperimenta: [https://github.com/bfabio/containers](https://github.com/bfabio/containers).
+
+I community lab per questa stagione terminano qui, ci prendiamo una meritata vacanza. Ma è solo un arrivederci, ci rivediamo dopo l’estate! 
+
+Nel frattempo, [guarda la registrazione](https://www.youtube.com/watch?v=SwbP7DA9eR0&t=1995s) di questo lab e scarica la presentazione in PDF.
```
Comment (bfabio) [inline _posts/it/2022-07-07-perchè-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md:22]:
```suggestion
Nel frattempo, [guarda la registrazione](https://www.youtube.com/watch?v=SwbP7DA9eR0) di questo lab e scarica la presentazione in PDF.
```

File: _posts/it/2022-07-07-perchè-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Per ricevere i prossimi aggiornamenti resta in contatto con noi su [Slack Developers](https://slack.developers.italia.it/)-, iscriviti alla nostra [mailing list](https://www.google.com/url?q=https://unisciti.developers.italia.it&sa=D&source=docs&ust=1657178342195309&usg=AOvVaw0nzfBX6MfvZfFSrdDIEyLU) e continua a seguirci sugli account Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```
Comment (bfabio) [inline _posts/it/2022-07-07-perchè-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md:24]:
```suggestion
Per ricevere i prossimi aggiornamenti resta in contatto con noi su [Slack Developers](https://slack.developers.italia.it/)-, iscriviti alla nostra [mailing list](https://unisciti.developers.italia.it) e continua a seguirci sugli account Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```

**Review by bfabio — CHANGES_REQUESTED** [review]

**Comment by bfabio** [comment]
@Francescopog I renamed the file so it doesn't contain accented characters, which look bad in the final URL

File: _posts/it/2022-07-07-perche-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Fabio ha illustrato i vantaggi dei container per lo sviluppo nell’ambito dei servizi pubblici, che contribuiscono a ridurre i problemi dati dalle differenze di ambienti, stanno alla base dei processi di *Continuous Integration e testing* e godono di un grande ecosistema in continua evoluzione.
+
+Come da tradizione nei nostri lab, Fabio ci ha portato velocemente dalla teoria alla pratica mostrandoci una demo di un container, usando lo strumento Docker e visualizzandone le principali caratteristiche. Insieme a lui, abbiamo usato un container già pronto, ne abbiamo creato uno nuovo e abbiamo infine combinato assieme più container.
+
+L’incontro si è chiuso con le domande dei partecipanti e con una call to action per tutti: Installa Docker o Podman e… sperimenta: [https://github.com/bfabio/containers](https://github.com/bfabio/containers).
+
+I community lab per questa stagione terminano qui, ci prendiamo una meritata vacanza. Ma è solo un arrivederci, ci rivediamo dopo l’estate!
+
+Nel frattempo, [guarda la registrazione](https://www.youtube.com/watch?v=SwbP7DA9eR0) di questo lab e [scarica la presentazione in PDF](/assets/images/posts/2022-07-07/290622_Community_lab.pdf).
```
Comment (bfabio) [inline _posts/it/2022-07-07-perche-container-sono-strumenti-utili-sviluppo-servizi.pubblici.md:22]:
```suggestion
Nel frattempo, **[guarda la registrazione](https://www.youtube.com/watch?v=SwbP7DA9eR0)** di questo lab e **[scarica la presentazione in PDF](/assets/images/posts/2022-07-07/290622_Community_lab.pdf)**.
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**